roles/pulp_api/README.md

```diff
@@ -8,7 +8,7 @@ Role Variables
 
 * `pulp_api_bind`: Interface and Port where Pulp Content [`gunicorn` service will
   listen.](https://docs.gunicorn.org/en/stable/settings.html#bind)
-  One can specify a unix socket path instead (recommended value is `'unix:/var/run/pulpcore-api/pulpcore-api.sock'`).
+  One can specify a unix socket path instead (recommended value is `'unix:/var/run/pulpcore-api.sock'`).
```

Comment:
I believe I picked my original value based on earlier example code. I believe the SELinux policy will cover this path as well, I am just not sure why you changed it:
https://github.com/pulp/pulpcore-selinux/blob/master/pulpcore.fc#L25

Also, on my laptop, the subfolder approach appears more common:
```
[mdepaulo@mdepaulo pulp_common]$ sudo find /var/run/ | grep sock
/var/run/docker/libnetwork/0bdad439ba73.sock
/var/run/docker/metrics.sock
/var/run/docker/containerd/containerd.sock
/var/run/docker/containerd/containerd.sock.ttrpc
/var/run/docker/containerd/containerd-debug.sock
/var/run/gssproxy.sock
/var/run/abrt/abrt.socket
/var/run/chrony/chronyd.sock
/var/run/libvirt/virtlogd-sock
/var/run/libvirt/virtlockd-sock
/var/run/libvirt/libvirt-sock-ro
/var/run/libvirt/libvirt-admin-sock
/var/run/libvirt/libvirt-sock
/var/run/docker.sock
/var/run/dbus/system_bus_socket
/var/run/avahi-daemon/socket
/var/run/cups/cups.sock
/var/run/lvm/lvmpolld.socket
```
Only docker (does their own thing a lot) and gssproxy (no subfolder, puts .pid file directly under /var/run also) do not use a subdir.

Reply:
That may have been my suggestion. Apache/nginx may not have permission to open `/var/run/pulpcore-api`. It's also why in the Foreman installer we chose the top level directory.